app.js

```diff
@@ -0,0 +1,29 @@
+const hamburgerIcon = document.querySelector('.image');
+const xIcon = document.querySelector('.image2');
+const navbar = document.querySelector('.navigator');
+const upper = document.querySelector('.upper');
+const linkMenu = document.querySelectorAll('.nav-link');
+
+hamburgerIcon.addEventListener('click', () => {
+  navbar.classList.toggle('menu-open');
+  upper.classList.toggle('upper-menu-open');
+  hamburgerIcon.classList.toggle('image-menu-open');
+  upper.classList.toggle('link-menu-open');
+  xIcon.classList.toggle('image-menu-appear');
+});
+
+xIcon.addEventListener('click', () => {
+  navbar.classList.remove('menu-open');
+  upper.classList.remove('upper-menu-open');
+  hamburgerIcon.classList.remove('image-menu-open');
+  upper.classList.remove('link-menu-open');
+  xIcon.classList.remove('image-menu-appear');
+});
+
+linkMenu.addEventListener('click', () => {
+  navbar.classList.remove('menu-open');
+  upper.classList.remove('upper-menu-open');
+  hamburgerIcon.classList.remove('image-menu-open');
+  upper.classList.remove('link-menu-open');
+  xIcon.classList.remove('image-menu-appear');
+});
```

Comment:
[REQUIRED] Nice implementation of the closing function. However, please note that this `event` will return and is actually returning an `error` stating that `Uncaught TypeError: linkMenu.addEventListener is not a function`. The reason is, you have selected a group of nodes (elements) when you declared:
```JAVASCRIPT
const linkMenu = document.querySelectorAll('.nav-link');
```
So, to make it work we need to access each node (element) of `linkMenu` and then add event to each of it. There number of ways of doing this, you are welcome to go with the option of your choice, as long as the `mob menu`  disappears when each link cliked, you will get approved. One of the simplest ways is to use `forEach()` method, please refer to the sample code below:

```JAVASCRIPT
```